src/main/java/com/github/igotyou/FactoryMod/utility/FactoryModGUI.java

```diff
@@ -168,6 +168,21 @@ private IClickable getSetupClick(FurnCraftChestEgg factory) {
 		return new LClickable(is, p -> showFactoryCreation(factory, true));
 	}
 
+	private IClickable getFuelClick(FurnCraftChestEgg factory, InputRecipe recipe) {
+		if (recipe == null) {
+			return null;
+		}
+		if (factory == null) {
+			return null;
+		}
+		ItemStack is = factory.getFuel();
+		ItemAPI.setDisplayName(is, ChatColor.GOLD + "Fuel cost for recipe");
+		List<String> lore = new ArrayList<>();
```

Comment:
`ItemAPI.addLore` makes this much shorter and will also be compatible with future fuel which has lore

Reply:
When using addLore, everytime I clicked the item it would append its lore again and again until off the screen